packages/discord.js/typings/index.d.ts

```diff
@@ -513,7 +513,7 @@ export type CategoryChannelResolvable = Snowflake | CategoryChannel;
 export abstract class Channel extends Base {
   public constructor(client: Client, data?: RawChannelData, immediatePatch?: boolean);
   public readonly createdAt: Date;
-  public readonly createdTimestamp: number;
+  public readonly createdTimestamp: number | null;
```

Comment:
Was this intentional? If yes, then `createdAt` would be nullable too

Reply:
true